### PR TITLE
[stable/kong] Kong prometheus servicemonitor label

### PR DESCRIPTION
**Pull request by sekka1**
Prometheus operator uses a label selector on the `ServiceMonitor` to determine which `ServiceMonitor` to pick up (https://github.com/coreos/kube-prometheus/pull/16#issuecomment-305933103).  The label can be custom to each prometheus-operators usage.  This PR adds the ability to add in a custom label on the `ServiceMonitor`.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by zanhsieh** [comment]
@sekka1 
Please resolve README.md conflict. Thank you.

**Comment by sekka1** [comment]
@zanhsieh fixed the conflict.

File: stable/kong/values.yaml [hunk 1]
```diff
@@ -438,3 +438,9 @@ serviceMonitor:
   # interval: 10s
   # Specifies namespace, where ServiceMonitor should be installed
   # namespace: monitoring
+  # The prometheus-operator selects scrape targets via a label on the ServiceMonitor,
+  # this label can be custom per the operators configuration.
+  prometheusOperatorServiceMonitorSelector:
+    enabled: false
+    matchLabels:
+      prometheus: somelabel
```
Comment (hbagdi) [inline stable/kong/values.yaml]: This looks very weird.
Why is this additional level of `prometheusOperatorServiceMonitorSelector` required?

I'd propose that we do something simpler like:

```yaml
serviceMonitor:
  enabled: false
  labels:
    foo: bar
```

Also, enabled for the labels  should not be necessary for labels.
You can simply check if any labels are set and inject them:
https://github.com/helm/charts/blob/master/stable/kong/templates/deployment.yaml#L31

**Review by hbagdi — CHANGES_REQUESTED** [review]

File: stable/kong/values.yaml (hunk 1 above)
Comment (sekka1) [reply stable/kong/values.yaml]: Good point =)....updating...

Comment (sekka1) [reply stable/kong/values.yaml]: @hbagdi updated.  This looks way better than my long variable name =)

**Review by hbagdi — APPROVED** [review]
/lgtm

**Pull request merged**